Author: Luca Boccassi <luca.boccassi@gmail.com>
Description: Remove dependency on conftest.sh from kernel modules
 The new build system of the Nvidia kernel modules now runs conftest.sh by
 having the modules objects depend on it. Simply overriding the script with a
 noop is no longer necessary, as the conftest/ directory with the headers will
 be created and will fail to build. As a workaround, simply take out the
 dependency from the modules, so that the build system doesn't even try to run
 the scripts and create the files.

--- a/nvidia-modeset/nvidia-modeset.Kbuild
+++ b/nvidia-modeset/nvidia-modeset.Kbuild
@@ -76,7 +76,7 @@
 # Register the conftests needed by nvidia-modeset.ko
 #
 
-NV_OBJECTS_DEPEND_ON_CONFTEST += $(NVIDIA_MODESET_OBJECTS)
+#NV_OBJECTS_DEPEND_ON_CONFTEST += $(NVIDIA_MODESET_OBJECTS)
 
 NV_CONFTEST_MACRO_COMPILE_TESTS += INIT_WORK
 NV_CONFTEST_TYPE_COMPILE_TESTS += file_inode
